SLHCUpgradeSimulations/Configuration/python/phase1TkCustoms.py

```diff
@@ -324,135 +324,138 @@ def customise_Reco(process,pileup):
-
-    # New tracking.  This is really ugly because it redefines globalreco and reconstruction.
-    # It can be removed if change one line in Configuration/StandardSequences/python/Reconstruction_cff.py
-    # from RecoTracker_cff.py to RecoTrackerPhase1PU140_cff.py
-
-    # remove all the tracking first
-    itIndex=process.globalreco_tracking.index(process.trackingGlobalReco)
-    grIndex=process.globalreco.index(process.globalreco_tracking)
-
-    process.globalreco.remove(process.globalreco_tracking)
-    process.globalreco_tracking.remove(process.iterTracking)
-    process.globalreco_tracking.remove(process.electronSeedsSeq)
-    process.reconstruction_fromRECO.remove(process.trackingGlobalReco)
-    process.reconstruction_fromRECO.remove(process.electronSeedsSeq)
-    process.reconstruction_fromRECO.remove(process.initialStepSeedLayers)
-    process.reconstruction_fromRECO.remove(process.initialStepSeeds)
-    process.reconstruction_fromRECO.remove(process.initialStepClassifier1)
-    process.reconstruction_fromRECO.remove(process.initialStepClassifier2)
-    process.reconstruction_fromRECO.remove(process.initialStepClassifier3)
-    process.reconstruction_fromRECO.remove(initialStepTrackCandidates)
-    process.reconstruction_fromRECO.remove(initialStepTracks)
-    process.reconstruction_fromRECO.remove(lowPtTripletStepClusters)
-    process.reconstruction_fromRECO.remove(lowPtTripletStepSeedLayers)
-    process.reconstruction_fromRECO.remove(lowPtTripletStepSeeds)
-    process.reconstruction_fromRECO.remove(lowPtTripletStep)
-    process.reconstruction_fromRECO.remove(lowPtTripletStepTrackCandidates)
-    process.reconstruction_fromRECO.remove(lowPtTripletStepTracks)
-
-    process.reconstruction_fromRECO.remove(mixedTripletStep)
-    process.reconstruction_fromRECO.remove(mixedTripletStepClusters)
-    process.reconstruction_fromRECO.remove(mixedTripletStepSeedLayersA)
-    process.reconstruction_fromRECO.remove(mixedTripletStepSeedLayersB)
-    process.reconstruction_fromRECO.remove(mixedTripletStepSeeds)
-    process.reconstruction_fromRECO.remove(mixedTripletStepSeedsA)
-    process.reconstruction_fromRECO.remove(mixedTripletStepSeedsB)
-    process.reconstruction_fromRECO.remove(mixedTripletStepClassifier1)
-    process.reconstruction_fromRECO.remove(mixedTripletStepClassifier2)
-    process.reconstruction_fromRECO.remove(mixedTripletStepTrackCandidates)
-    process.reconstruction_fromRECO.remove(mixedTripletStepTracks)
-
-    process.reconstruction_fromRECO.remove(pixelPairStepClusters)
-    process.reconstruction_fromRECO.remove(pixelPairStepSeeds)
-    process.reconstruction_fromRECO.remove(pixelPairStepSeedLayers)
-    process.reconstruction_fromRECO.remove(pixelPairStep)
-    process.reconstruction_fromRECO.remove(pixelPairStepTrackCandidates)
-    process.reconstruction_fromRECO.remove(pixelPairStepTracks)
+    if not eras.trackingPhase1.isChosen():

```

Comment:
Well, it's not yet (as in this PR) completely deprecated :) I could add (in a separate PR) a deprecation warning to the `customise()` (in this file) if it is called without `phase1Pixel` era set.